### PR TITLE
New Screenshot from Secrid

### PR DESCRIPTION
**Pull request by ftiff**
@mikevos from Secrid was kind enough to do a screenshot of SplashBuddy in action!
I also removed the dead button for donation and updated badge version to 1.2

### CODE REVIEW

**Comment by Ethenyl** [comment]
Hi @ftiff, changes are valid but can you target the branch `feature/migration-to-amaris`?

The actual code in master is still **v1.1**.

Or we can already merge the other branch and your branch after, as you prefer.

**Comment by ftiff** [comment]
Changed!

File: README.md [hunk 1]
```diff
@@ -4,9 +4,8 @@
 ![](https://img.shields.io/badge/macOS-10.13-brightgreen.svg)
 
 
-![](https://img.shields.io/badge/Version-1.0-lightgrey.svg)
+![](https://img.shields.io/badge/Version-1.2-lightgrey.svg)
```
Comment (Ethenyl) [inline README.md]: Go with 1.2.0 instead, more compliant with **Apple** versioning system.

**Comment by ftiff** [comment]
@Ethenyl I let you merge it if it's ok for you!

**Pull request merged**